api.md

```diff
@@ -2041,6 +2043,36 @@ SELECT * FROM show_tablespaces('conditions');
 
 ---
 
+## timescaledb_pre_restore() [](timescaledb_pre_restore)
+
+Perform the proper operations to allow restoring of the database via `pg_restore` to commence.
+Specifically this sets the `timescaledb.restoring` GUC to `on` and stops any
```

Comment:
"...stops (pauses?) any running background workers until..."

Reply:
Stops is correct here I think.

Reply:
So to clarify:  Does it kill the background worker, such that if it's in the middle of a txn, the txn will be aborted?  If so, I think "stops" is appropriate.

Reply:
yes.